Realm/RLMAccessor.h

```diff
@@ -44,11 +44,11 @@ bool RLMIsGeneratedClass(Class cls);
 // Dynamic getters/setters
 //
 FOUNDATION_EXTERN void RLMDynamicValidatedSet(RLMObjectBase *obj, NSString *propName, id __nullable val);
-FOUNDATION_EXTERN RLMProperty *RLMValidatedGetProperty(RLMObjectBase *obj, NSString *propName);
 FOUNDATION_EXTERN id __nullable RLMDynamicGet(RLMObjectBase *obj, RLMProperty *prop);
+FOUNDATION_EXTERN id __nullable RLMDynamicGetByName(RLMObjectBase *obj, NSString *propName, bool asList);
```

Comment:
This broke RealmSwift (Object.swift line 252).
(Not sure if outsiders are supposed to post comments on PRs so I apologize if I'm overstepping a boundary here).